CONTRIBUTING.md

```diff
@@ -117,7 +121,31 @@ npm run refresh-package-lock-files
 # Finally, check that the dependency in question has been updated in `packages/shared-metrics/package-lock.json` to the desired version.
 ```
 
-Note: The intent of the procedure above is satisfy security scanners (`npm audit`, Dependabot etc.). `package-lock.json` files are never included when publishing to the npm registry. Thy are only relevant when you execute an `npm install` in _this repository_. Users of `@instana` npm packages will have their own `package-lock.json` or `yarn.lock` file for their application, those are completely independent from the `package-lock.json` files in this repository. Thus, they need to take care of updating their transitive dependencies themselves. We only can make sure that the version ranges we use for our direct dependencies allow to install a dependency tree that has no known vulnerabilities.
```

Comment:
Most of this has been moved up, so we now only link to the section where this is explained.